modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -22,17 +22,16 @@
 	alt_titles = list("Salon Manager", "Salon Technician", "Stylist", "Colorist")
 
 /datum/job/bartender
-	alt_titles = list("Bartender", "Mixologist")
+	alt_titles = list("Bartender", "Mixologist", "Barkeeper")
 
 /datum/job/blueshield
 	alt_titles = list("Blueshield", "Command Bodyguard", "Executive Protection Agent")
 
-
 /datum/job/botanist
 	alt_titles = list("Botanist", "Hydroponicist", "Gardener", "Botanical Researcher", "Herbalist")
 
 /datum/job/bouncer
-
+	alt_titles = list("Bouncer", "Doorman", "Door Supervisor")
```

Comment:
:-1:
I'd only really want to add Service Guard to the list here; bouncing the bar is only part of their job.

Reply:
fair, hard to get titles for them